.github/workflows/sync.yml

```
@@ -0,0 +1,18 @@
+name: Sync Files
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  convert-test-sync:
+    name: Sync PCL test files
+    runs-on: macos-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Run GitHub File Sync
+        uses: BetaHuhn/repo-file-sync-action@v1
+        with:
+          GH_PAT: ${{ secrets.PULUMI_ACCESS_TOKEN }}
```

Close! GH_PAT is asking for a GitHub personal access token, not a Pulumi access token.

This should work:

```suggestion
          GH_PAT: ${{ secrets.PULUMI_BOT_TOKEN }}
```